doc/user-guide/user_guide.md

```diff
@@ -92,25 +62,31 @@ CREATE JAVA ADAPTER SCRIPT SCHEMA_FOR_VS_SCRIPT.JDBC_ADAPTER_SCRIPT AS
 /
 ```
 
-## Using the Adapter
 
-The following statements demonstrate the common steps to create a Virtual Schema. 
-You can find the similar tutorial with dialect-specific information for each dialect in the [supported dialects list](#supported-dialects). 
+## Define a named connection.
 
-Please scroll down to see a list of all properties supported by the JDBC adapter.
-
-First we create a connection and then a virtual schema. The adapter will retrieve the metadata via JDBC and map them to virtual tables. 
-The metadata (virtual tables, columns and data types) are then cached in Exasol.
+First we create a connection and then a virtual schema.
```

Comment:
Remove here a virtual schema mentioning as you divided them into two sections

Reply:
```suggestion
First we create a connection.
```